guide/react-native/screensharing.mdx

```diff
@@ -0,0 +1,192 @@
+---
+sidebar_position: 5
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Screen sharing
+
+Our SDK also allow to stream content of mobile device screen.
+
+## Installation
+
+### Android
+
+On Android devices, screen share feature is available right after SDK is installed in app. No further actions are required.
```

Comment:
**issue:** This is not true. Screen sharing won't work on Android >=24 if you don't enable `enableForegroundService`, as we need a foreground service for that to work.